reviews/utils/review_utils.py

```diff
@@ -431,3 +430,45 @@ def discontinue_review(review):
     review.date_end = datetime.datetime.now()
     review.save()
     review.proposal.save()
+
+
+def assign_reviewers(review, list_of_users, route):
+    """Assigns or reassigns a list of reviewers to a review"""
+
+    # Set stage to commission if users are assigned
+    if len(list_of_users) > 0:
+        review.stage = review.COMMISSION
+    else:
+        review.stage = review.ASSIGNMENT
```

Comment:
Does this actually happen?

The assignment form has this validation rule:
```python
    def clean_reviewers(self):
        reviewers = self.cleaned_data['reviewers']

        if len(reviewers) == 0:
            raise ValidationError(
                _('Er moet tenminste één beoordelaar geselecteerd worden.'),
                code='no_reviewer')

        # See PR #188 if a secretary should be added to every review
        return self.cleaned_data['reviewers']
```

Reply:
Nope, this should never happen under normal circumstances. But when moving it into the utils package I took the liberty of adding this to make the function more utility-like, should we ever want to bump an application back to the assignment phase manually.

Reply:
That makes sense, good thinking